src/main/java/com/commercetools/emailprocessor/model/ProjectConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package com.commercetools.emailprocessor.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProjectConfiguration {
+    private static final Logger LOG = LoggerFactory.getLogger(ProjectConfiguration.class);
+    private List<TenantConfiguration> tenants;
+
+    public ProjectConfiguration() {
+    }
+
+    public ProjectConfiguration(@Nonnull final List<TenantConfiguration> currentTenants) {
+        this.tenants = currentTenants;
+    }
+
+    public List<TenantConfiguration> getTenants() {
+        return tenants;
+    }
+
+    public void setTenants(@Nonnull final List<TenantConfiguration> tenants) {
```

Comment:
the class must be immutable - remove this setter

Reply:
I need the setter, because the jsonmapper use then to create this object

Reply:
As i remember, Jackson uses private properties over reflection, so public setters are not needed

Reply:
done

Reply:
1fadcd0